eng/scripts/Create-Apiview-Token-Cadl.ps1

```diff
@@ -78,7 +79,13 @@ if ($revs)
         # initialize git clone if current review is generated from different repo than previous one
         if ($GitRepoName -ne $prevRepo)
         {
-            git clone --no-checkout --filter=tree:0 "https://github.com/$GitRepoName"
+            $gitUrl = "https://github.com"
+            if ($GitPat)
+            {
+                $gitUrl = "https://$GitPat@github.com"
+            }
+
+            git clone --no-checkout --filter=tree:0 "$gitUrl/$GitRepoName"

```

Comment:
Not introduced with this change but this multiple clone in the same directory might cause issues with multiple reviews.